### PR TITLE
Gtfs pipeline draft

### PR DESCRIPTION
**Pull request by yehudalev**
I just want to check out what is the problem, it throws errors when I run the command 
`bazel build //projects/opentrain/gtfs_pipeline:data_analysis_part1`

I get the errors: 
`INFO: Invocation ID: 386f48aa-f784-4e52-a515-eb1e9be667f4
INFO: SHA256 (https://firebasestorage.googleapis.com/v0/b/startupos-5f279.appspot.com/o/israel-public-transportation.zip?alt=media&token=a9bc43a5-36a6-4126-9e19-2e42f9ff663c) = 3c44446ffeaea51188a284484553623543d03b94695f3a363131bb3eda86e1c2
ERROR: /home/dd/hasadna/projects/opentrain/gtfs_pipeline/BUILD:3:1: no such package '@gtfs_data//': java.io.IOException: Prefix input_gtfs_data was given, but not found in the zip and referenced by '//projects/opentrain/gtfs_pipeline:data_analysis_part1'
ERROR: /home/dd/hasadna/projects/opentrain/gtfs_pipeline/BUILD:3:1: no such package '@gtfs_data//': java.io.IOException: Prefix input_gtfs_data was given, but not found in the zip and referenced by '//projects/opentrain/gtfs_pipeline:data_analysis_part1'
ERROR: /home/dd/hasadna/projects/opentrain/gtfs_pipeline/BUILD:3:1: no such package '@gtfs_data//': java.io.IOException: Prefix input_gtfs_data was given, but not found in the zip and referenced by '//projects/opentrain/gtfs_pipeline:data_analysis_part1'
ERROR: /home/dd/hasadna/projects/opentrain/gtfs_pipeline/BUILD:3:1: no such package '@gtfs_data//': java.io.IOException: Prefix input_gtfs_data was given, but not found in the zip and referenced by '//projects/opentrain/gtfs_pipeline:data_analysis_part1'
ERROR: /home/dd/hasadna/projects/opentrain/gtfs_pipeline/BUILD:3:1: no such package '@gtfs_data//': java.io.IOException: Prefix input_gtfs_data was given, but not found in the zip and referenced by '//projects/opentrain/gtfs_pipeline:data_analysis_part1'
ERROR: /home/dd/hasadna/projects/opentrain/gtfs_pipeline/BUILD:3:1: no such package '@gtfs_data//': java.io.IOException: Prefix input_gtfs_data was given, but not found in the zip and referenced by '//projects/opentrain/gtfs_pipeline:data_analysis_part1'
ERROR: /home/dd/hasadna/projects/opentrain/gtfs_pipeline/BUILD:3:1: no such package '@gtfs_data//': java.io.IOException: Prefix input_gtfs_data was given, but not found in the zip and referenced by '//projects/opentrain/gtfs_pipeline:data_analysis_part1'
ERROR: /home/dd/hasadna/projects/opentrain/gtfs_pipeline/BUILD:3:1: no such package '@gtfs_data//': java.io.IOException: Prefix input_gtfs_data was given, but not found in the zip and referenced by '//projects/opentrain/gtfs_pipeline:data_analysis_part1'
ERROR: /home/dd/hasadna/projects/opentrain/gtfs_pipeline/BUILD:3:1: no such package '@gtfs_data//': java.io.IOException: Prefix input_gtfs_data was given, but not found in the zip and referenced by '//projects/opentrain/gtfs_pipeline:data_analysis_part1'
ERROR: /home/dd/hasadna/projects/opentrain/gtfs_pipeline/BUILD:3:1: no such package '@gtfs_data//': java.io.IOException: Prefix input_gtfs_data was given, but not found in the zip and referenced by '//projects/opentrain/gtfs_pipeline:data_analysis_part1'
ERROR: Analysis of target '//projects/opentrain/gtfs_pipeline:data_analysis_part1' failed; build aborted: no such package '@gtfs_data//': java.io.IOException: Prefix input_gtfs_data was given, but not found in the zip
INFO: Elapsed time: 60.170s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
    Fetching @gtfs_data; fetching 59s`

thanks

`

### CODE REVIEW

File: projects/opentrain/gtfs_pipeline/handle_data/Main.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.IntStream;
+
+import sun.awt.windows.WLightweightFramePeer;
+
+class Handle_data
```
Comment (val-fed) [inline projects/opentrain/gtfs_pipeline/handle_data/Main.java]: The file name (`Main.java`) and the actual class name in the code(`Handle_data`) must be the same. Class names should be nouns, in mixed case with the first letter of each internal word capitalized. 
You can read more about Java naming convention here - https://www.oracle.com/technetwork/java/codeconventions-135099.html

I suggest naming this class `DataHandler`.
So, please do:
- Rename `Main.java` file to `DataHandler.java`
- s / Handle_data / DataHandler

File: projects/opentrain/gtfs_pipeline/handle_data/BUILD [hunk 2]
```diff
@@ -0,0 +1,6 @@
+java_binary(
+    name = "handle_data",
+    srcs = ["Main.java"],
```
Comment (val-fed) [inline projects/opentrain/gtfs_pipeline/handle_data/BUILD]: s / Main.java / DataHandler.java

File: projects/opentrain/gtfs_pipeline/handle_data/Main.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.IntStream;
+
+import sun.awt.windows.WLightweightFramePeer;
```
Comment (val-fed) [inline projects/opentrain/gtfs_pipeline/handle_data/Main.java]: Remove unused imports:
- import sun.awt.windows.WLightweightFramePeer;
- import java.util.stream.IntStream;

File: projects/opentrain/gtfs_pipeline/handle_data/Main.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import java.io.IOException;
```
Comment (val-fed) [inline projects/opentrain/gtfs_pipeline/handle_data/Main.java]: Add `package projects.opentrain.gtfs_pipeline.handle_data;` definition. It should be the before imports.

File: projects/opentrain/gtfs_pipeline/handle_data/BUILD [hunk 5]
```diff
@@ -0,0 +1,6 @@
+java_binary(
+    name = "handle_data",
+    srcs = ["Main.java"],
+    main_class = "handle_data",
```
Comment (val-fed) [inline projects/opentrain/gtfs_pipeline/handle_data/BUILD]: s / "handle_data" / "projects.opentrain.gtfs_pipeline.handle_data.DataHandler"

File: projects/opentrain/gtfs_pipeline/handle_data/Main.java (hunk 1 above)
Comment (yehudalev) [reply projects/opentrain/gtfs_pipeline/handle_data/Main.java]: > The file name (`Main.java`) and the actual class name in the code(`Handle_data`) must be the same.
Just curiuos, in the example `data_analysis/task1/Main.java` the class in `Main.java` is `Task1`, is there a spacial case there?

File: projects/opentrain/gtfs_pipeline/handle_data/Main.java (hunk 4 above)
Comment (yehudalev) [reply projects/opentrain/gtfs_pipeline/handle_data/Main.java]: Do you mean package projects.opentrain.gtfs_pipeline.*DataHandler*;

Comment (val-fed) [reply projects/opentrain/gtfs_pipeline/handle_data/Main.java]: Just `package projects.opentrain.gtfs_pipeline.handle_data;`, without `DataHandler`.

File: projects/opentrain/gtfs_pipeline/handle_data/Main.java (hunk 1 above)
Comment (val-fed) [reply projects/opentrain/gtfs_pipeline/handle_data/Main.java]: Yes, you are right. Sorry about it.
It's possible to have different names. You can choose the option from examples.

Just take attention to it.
```
java_binary(
    name = "task1",
    srcs = ["Main.java"],
    deps = ["//projects/data_analysis/common:java_common", "//third_party/maven/com/google/protobuf:protobuf_java"],
    main_class = "com.example.tasks.Task1",
    visibility = ["//projects/data_analysis:__pkg__"]
)
```
`srcs = ["Main.java"]` - name of the file
`main_class = "com.example.tasks.Task1"` -  package + class name inside `Main.java`

Comment (oferb) [reply projects/opentrain/gtfs_pipeline/handle_data/Main.java]: It's actually a mistake in `data_analysis`. I mean, it compiles, but the file should be named `Task1.java`.

**Pull request closed without merging**